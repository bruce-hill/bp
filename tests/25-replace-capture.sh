# Captures with @ can be referenced in a replacement by @1, @2, etc.
# Example: bp -p '"=" _ @+`0-9 => "= -@1"' replaces "x = 5" with "x = -5"
# Note: @0 refers to the entire match, e.g. bp -p '"foo" => "xx@0xx"' replaces "foo" with "xxfooxx"
bp -p '@`a,e,i,o,u => "{@1}" / .'
