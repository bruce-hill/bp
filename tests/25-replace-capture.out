{o}n{e}
tw{o}
thr{e}{e}
f{o}{u}r
f{i}v{e}
